examples/calls/delete_call_log.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
```

Comment:
Lets move down the import statement

Reply:
Addressed in [474f18f](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/474f18fdb6516b8ff29c450126cde2c618369527)